sympy/functions/special/zeta_functions.py

```diff
@@ -517,3 +517,69 @@ def eval(cls, s):
 
     def _eval_rewrite_as_zeta(self, s):
         return (1 - 2**(1 - s)) * zeta(s)
+
+
+class stieltjes(Function):
+    """
+    Represents Stieltjes constants, :math: `\gamma_{k}` that occur in Laurent
+    Series expansion of the Riemann zeta function.
+
+    Examples
+    ========
+
+    >>> from sympy import stieltjes
+    >>> from sympy.abc import n
+
```

Comment:
Remove this line as it splits the code block generated in the doc. Same for all the following cases.